diff --git a/node_modules/vuetify/dist/vuetify.js b/node_modules/vuetify/dist/vuetify.js
index b00221a..6a261c8 100644
--- a/node_modules/vuetify/dist/vuetify.js
+++ b/node_modules/vuetify/dist/vuetify.js
@@ -15192,10 +15192,7 @@ var __assign = undefined && undefined.__assign || function () {
   render: function render(h) {
     return h('div', {
       staticClass: 'v-expansion-panel',
-      class: this.classes,
-      attrs: {
-        'aria-expanded': String(this.isActive)
-      }
+      class: this.classes
     }, Object(_util_helpers__WEBPACK_IMPORTED_MODULE_2__["getSlot"])(this));
   }
 }));
diff --git a/node_modules/vuetify/lib/components/VExpansionPanel/VExpansionPanel.js b/node_modules/vuetify/lib/components/VExpansionPanel/VExpansionPanel.js
index 17c42b8..563af5e 100644
--- a/node_modules/vuetify/lib/components/VExpansionPanel/VExpansionPanel.js
+++ b/node_modules/vuetify/lib/components/VExpansionPanel/VExpansionPanel.js
@@ -73,10 +73,7 @@ export default mixins(GroupableFactory('expansionPanels', 'v-expansion-panel', '
   render(h) {
     return h('div', {
       staticClass: 'v-expansion-panel',
-      class: this.classes,
-      attrs: {
-        'aria-expanded': String(this.isActive)
-      }
+      class: this.classes
     }, getSlot(this));
   }
 
